src/js/helpers/awardAmountHelper.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * awardAmountHelper.js
+ * Created by michaelbray on 3/7/17.
+ */
+
+import * as MoneyFormatter from 'helpers/moneyFormatter';
+
+export const formatAwardAmountRange = (range) => {
+    const min = MoneyFormatter.calculateUnitForSingleValue(range[0]);
+    const max = MoneyFormatter.calculateUnitForSingleValue(range[1]);
+
+    const minValue = Math.round((10 * range[0]) / min.unit) / 10;
+    const maxValue = Math.round((10 * range[1]) / max.unit) / 10;
+
+    const minLabel = `${minValue}${min.unitLabel}`;
+    const maxLabel = `${maxValue}${max.unitLabel}`;
+
+    if (range[0] === 0 && range[1] === 0) {
+        return `$0 & Above`;
+    }
+    else if (range[0] === 0) {
+        return `Under $${maxLabel}`;
+    }
+    else if (range[1] === 0) {
+        return `$${minLabel} & Above`;
+    }
+    return `$${minLabel} - $${maxLabel}`;
+};
+
+export const ensureInputIsNumeric = (input) => {
+    // Remove non-numeric characteres from input
+    const cleanInput = (input.toString()).replace(/\D/g, '');
```

Comment:
Unfortunately this also strips out decimal points, so `1.2` becomes `12`. AccountingJS has an [unformat()](http://openexchangerates.github.io/accounting.js/#documentation) function that may be helpful.